### PR TITLE
update check proxy version

### PR DESCRIPTION
**Pull request by hzxuzhonghu**

### CODE REVIEW

File: pilot/pkg/networking/util/util.go [hunk 1]
```diff
@@ -212,20 +212,13 @@ func SortVirtualHosts(hosts []route.VirtualHost) {
...
-	return isProxyVersion(node, "1.")
-}
-
-// Is11Proxy checks whether the given Proxy version is 1.1.
-func Is11Proxy(node *model.Proxy) bool {
-	return isProxyVersion(node, "1.1")
+// IsProxyVersionGE11 checks whether the given Proxy version is greater than or equals 1.1.
+func IsProxyVersionGE11(node *model.Proxy) bool {
+	ver, _ := node.GetProxyVersion()
+	if ver >= "1.1" {
```
Comment (rshriram) [inline pilot/pkg/networking/util/util.go]: do strings have arithmetic comparison operators like this?

Comment (hzxuzhonghu) [reply pilot/pkg/networking/util/util.go]: yes, https://golang.org/src/strings/compare.go

File: pilot/pkg/networking/util/util_test.go [hunk 2]
```diff
@@ -97,12 +97,11 @@ func TestGetNetworkEndpointAddress(t *testing.T) {
 	}
 }
 
-func Test_isProxyVersion(t *testing.T) {
+func TestIsProxyVersionGE11(t *testing.T) {
```
Comment (hzxuzhonghu) [inline pilot/pkg/networking/util/util_test.go]: Also here is the unit test .

**Pull request merged**